tests/unit/components/UI/Alert/ImageUploaderDialog.spec.js

```diff
@@ -0,0 +1,29 @@
+import { mount } from "@vue/test-utils";
+import store from "@/store";
```

Comment:
Why import store here? It's already imported globally right?

Reply:
that is for using `this.$store`. this is for accessing actions and state. will try setting it globally too.